### ISSUE TITLE
Outputting value from YAML file to stdout has trailing ellipsis

### ISSUE THREAD

**Issue by twolfson** [mozilla/sops#119]
We are using SOPS to dump specific values in some scripts. Unfortunately, SOPS seems to be adding a `...` to the end of the output. Here are steps to reproduce:

``` bash
$ # Generate a temporary PGP backed file
$ sops tmp.yml --pgp 'XXXX'
INFO: tmp.yml doesn't exist, creating it.
INFO: generating and storing data encryption key
INFO: temp file created at /tmp/tmpJC0MRr.yaml
INFO: file written to tmp.yml
$ # Output content in plaintext
$ sops tmp.yml --decrypt
gpg: encrypted with 2048-bit RSA key, ID XXXX, created 2016-09-09
      "XXXX"
foo: bar
$ # Output value on leaf of YAML tree
$ sops tmp.yml --decrypt --extract '["foo"]'
gpg: encrypted with 2048-bit RSA key, ID XXXX, created 2016-09-09
      "XXXX"
bar
...
```

After doing some digging, it looks like this is due to us doing an `fd` check along side `isinstance(tree, list)/isinstance(tree, dict)` when we shouldn't. As a result, we are dumping a value as YAML when it's not:

https://github.com/mozilla/sops/blob/1.14/sops/__init__.py#L1281-L1285

https://github.com/mozilla/sops/blob/1.14/sops/__init__.py#L1287-L1292

**Comment by twolfson** [comment]
I'll be submitting a PR to patch this shortly

**Comment by twolfson** [comment]
This was resolved via #120
